ln-simln-jamming/src/attack_interceptor.rs

```diff
@@ -221,8 +79,8 @@ where
                 records_from_endorsement(EndorsementSignal::Endorsed);
         }
 
-        // The target is not involved in the forward at all, just use jamming interceptor to implement reputation
-        // and bucketing.
+        // If attacker is not involved, use jamming interceptor to implement reputation and
+        // bucketing.
```

Comment:
We can now remove the hard-set of `incoming_custom_records` - previously we were relying on `intercept_peer_outgoing` to filter out anything that wasn't upgradable and then setting endorsed here (because we know it must be upgradable).

This can now purely be handled by the `reputation_interceptor`! (and we get to delete moar code 😻 )